.github/workflows/CD.yml

```diff
@@ -0,0 +1,20 @@
+name: CD
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
I think there's a recent version of this action.

Reply:
Of course there’s is a version 4. Make sure to use the latest versions of the actions you’re using.